web_src/js/features/comp/EditorMarkdown.test.ts

```diff
@@ -32,10 +32,10 @@ test('EditorMarkdown', () => {
   testInput({value: '1. \n2. ', pos: 3}, {value: '\n2. ', pos: 0});
 
   testInput('- x', '- x\n- ');
-  testInput('1. foo', '1. foo\n1. ');
-  testInput({value: '1. a\n2. b\n3. c', pos: 4}, {value: '1. a\n1. \n2. b\n3. c', pos: 8});
+  testInput('1. foo', '1. foo\n2. ');
+  testInput({value: '1. a\n2. b\n3. c', pos: 4}, {value: '1. a\n2. \n2. b\n3. c', pos: 8});
```

Comment:
The more background details are here:

* https://github.com/go-gitea/gitea/pull/32966#issuecomment-2560499604
* https://github.com/go-gitea/gitea/pull/32966#issuecomment-2560837047

I didn't do that because there will be many edge cases to make the "increment" right. While keep using `1. ` still make the render work right.

----

So here, I am not sure `1 2 2 3` is better or ` 1 1 1 1` is better.